src/routes/index.svelte

```diff
@@ -138,12 +138,24 @@
         }
     }
 
-    function shareResults() {
+    function shareResults(e) {
```

Comment:
e is never being passed from anywhere? Also i agree with vco here h is superior

Reply:
> e is never being passed from anywhere? Also i agree with vco here h is superior


It is passed from the JavaScript click event. "e" is shorthand for "event", referring to the JavaScript Event object ([Mozilla Documentation](https://developer.mozilla.org/en-US/docs/Web/API/Event)).